docs/concepts/emails.mdx

```diff
@@ -9,15 +9,98 @@ Per default, Ory Cloud has the SMTP email sending configured. Ory Cloud sends Em
 sending server (`connection_uri`).
+
+To configure these values with the Ory CLI use:
+
+```shell
+ory patch project {your-project-id} \
+  --replace '/services/identity/config/courier/smtp/connection_uri="smtp://username:password@server:port/"' \
+  --replace '/services/identity/config/courier/smtp/from_address="hello@example.org' \
+  --replace '/services/identity/config/courier/smtp/from_name="My Company"'
+```
+
+### SMTP Connection URI
+
+SMTP has six different security mechanisms. Most SMTP services today use Explicit StartTLS with trusted certificates - the item in
+the list below.
+
+1. **Recommended**: StartTLS with certificate trust verification. This is the most common option today:
+   ```
+   smtp://username:password@server:port/
+   ```
+1. StartTLS without certificates trust verification:
+   ```
+   smtp://username:password@server:port/?skip_ssl_verify=true
+   ```
+1. Cleartext SMTP uses no encryption and is not secure. This option is often used in development environments:
+   ```
+   smtp://username:password@server:port/?disable_starttls=true
+   ```
+1. Implicit TLS with certificates trust verification:
+   ```
+   smtps://username:password@server:port/
+   ```
+1. Implicit TLS without certificates trust verification:
+   ```
+   smtps://username:password@server:port/?skip_ssl_verify=true
+   ```
+1. Implicit TLS with certificate verification which works if the server is hosted on a sudomain and uses a non-wildcard domain
+   certificate:
+   ```
+   smtps://username:password@subdomain.my-mailserver.com:1234/?server_name=my-mailserver.com
+   ```
+
+#### Mailgun
+
+Use the following connection URI to
+[send emails using Mailgun](https://documentation.mailgun.com/en/latest/quickstart-sending.html):
+
+```
+smtp://<smtp-user>:<smtp-password>@smtp.mailgun.org:587
+
+# For example:
+# smtp://some-user%40mailgun.example.org:df2a2c4e-5caa-4f04-85b9-72d54a2468ad@smtp.eu.mailgun.org:587
+```
+
+#### AWS SES SMTP
+
+Use the following connection URI to
+[send emails using AWS SES SMTP](https://docs.aws.amazon.com/ses/latest/dg/send-email-smtp.html):
+
+```
+smtp://<smtp-user>:<smtp-password>@email-smtp.<region>.amazonaws.com:587/
+
+# For example:
+# smtp://theuser:the-password@email-smtp.eu-central-1.amazonaws.com:587/
+```
+
+#### Postmark
+
+Use the following connection URI to [send emails using Postmark](https://postmarkapp.com/smtp-service):
+
+```
+smtp://<YOUR_POSTMARK_SEVER_API_TOKEN>:<YOUR_POSTMARK_SEVER_API_TOKEN>@smtp.postmarkapp.com:587/
+
+# For example:
+# smtp://thetoken:thetoken@smtp.postmarkapp.com:587/
+```
+
+### Error Transparency
+
+In the future, we will improve error observability and transparency for this feature. If you have problems with email delivery
+over custom SMTP servers, please, for the time being, reach out to us on [Slack](https://slack.ory.sh) or
+[GitHub](https://github.com/ory/cloud) for assistance.
```

Comment:
```suggestion
### Troubleshooting

If you have problems setting up email delivery or you experience errors or bugs, reach out to the Ory team directly:

- Send us a message on [Slack](https://slack.ory.sh)
- Create an issue on [GitHub](https://github.com/ory/cloud)
```

Just a reminder - promising things in the documentation isn't a good practice. The docs should focus on the current state of things. "In the future" can mean different things to different people, including us. Therefore I extracted the important message from this paragraph and gave it a more suitable (IMO) heading. ;)

Reply:
Makes sense, the thinking here being that this state is really not acceptable (errors should be observable for our customers) and that we know that -> fixing it.

Mabye we can reference the issue? https://github.com/ory/cloud/issues/117

Reply:
@aeneasr I edited the suggestion to include a note that references the RPC issue. 👍🏻 or 👎🏻 ?